### ISSUE TITLE
Feature suggestion: colour gradient

### ISSUE THREAD

**Issue by TomHartleyYork** [fogleman/primitive#35]
I find these images hypnotic. I would like to see one composed of shapes with an RGB colour gradient fill. In many cases one shape (e.g., rectangle) with a gradient fill could outperform many single colour shapes. 

So fewer parameters needed to fit the same image.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices.  Legal places will deal with it. 

Some things which not obey rules which bad effected itself.

**Comment by TingYiShun** [comment]
That's true. There a re some kind of shapes which cannot changed. Ex, places with mechanic or with some high tech-devices. Legal places will deal with it.

Some things which not obey rules which bad effected itself.

**Comment by seanjensengrey** [comment]
![over_gradient](https://user-images.githubusercontent.com/128454/32813551-b41a5ee8-c95f-11e7-86cd-42ca99079764.png)

from https://medium.freecodecamp.org/using-svg-as-placeholders-more-image-loading-techniques-bed1b810ab2c

seems like an idea candidate for replacing lots of primitives with a gradient across a primitive.
